editor/plugins/polygon_2d_editor_plugin.h

```diff
@@ -127,7 +126,7 @@ class Polygon2DEditor : public AbstractPolygon2DEditor {
 	Vector<int> polygon_create;
 	UVMode uv_move_current;
 	Vector2 uv_drag_from;
-	bool updating_uv_scroll;
+	bool updating_zoom;
```

Comment:
This can be removed. Use `set_value_no_signal()` or `set_block_signals()` on the scrollbars.

Reply:
Thanks for the tip, I didn't know the existence of these methods!

Reply:
And with this change this pr actually has sum of -1 LOC, nice! 😁